### PR TITLE
Issue#1

### PR DESCRIPTION
**Pull request by lucianozapata**
Started implementing #1 .

### CODE REVIEW

File: src/main/java/com/group24/decide/Decide.java [hunk 1]
```diff
@@ -36,16 +39,23 @@ public static void main(String[] args){
 
         Datapoints[] Points = new Datapoints[NUMPOINTS];
         Points[0] = new Datapoints(1, 2); // Input the data points
+        Points[0] = new Datapoints(4, 7); 
+        Points[0] = new Datapoints(7, 6); 
+        Points[0] = new Datapoints(9, 8); 
+        Points[0] = new Datapoints(10,20);
```
Comment (persman96) [inline src/main/java/com/group24/decide/Decide.java:45]: These should be Points[1], Points[2] etc.

**Review by persman96 — CHANGES_REQUESTED** [review]

File: src/main/java/com/group24/decide/LIC.java [hunk 2]
```diff
@@ -38,33 +37,16 @@ public LIC(Parameter parameters, Datapoints[] points){
...
-           double datapoint2Y = this.datapoints[index+1].y;
-
-           // TODO: Use a help function to calculate this distance!
-           if((sqrt(pow(datapoint1X - datapoint2X, 2) + pow(datapoint1Y -datapoint2Y, 2)) < this.parameters.LENGTH1)) {
-                return true;
-            }
-
-        }
-        return false;
-
```
Comment (persman96) [inline src/main/java/com/group24/decide/LIC.java]: I think you accidentally delete this.

Comment (lucianozapata) [reply src/main/java/com/group24/decide/LIC.java]: Yep, fixed it in next commit!

**Review by persman96 — APPROVED** [review]
Looks good!

**Review by nikolim — APPROVED** [review]
LGTM :+1:

**Pull request merged**